skiplang/prelude/ts/src/sk_runtime.ts

```diff
@@ -123,11 +123,11 @@ class LinksImpl implements Links {
 
     this.SKIP_js_get_argc = () => utils.args.length;
     this.SKIP_js_get_argn = (index: int) =>
-      utils.exportString(utils.args[index]);
+      utils.exportString(utils.args[index]!);
```

Comment:
Is the `index` guaranteed to be within bounds here?

Reply:
Fixed in new version.